modules/imgproc/test/test_goodfeaturetotrack.cpp

```diff
@@ -91,9 +91,9 @@ test_cornerEigenValsVecs( const Mat& src, Mat& eigenv, int block_size,
     denom = denom * denom;
 
     if( _aperture_size < 0 )
-        denom *= 4;
+        denom *= 2.;
     if(type != ftype )
-        denom *= 255.;
+        denom *= 255. * 255.;
```

Comment:
why?

Reply:
@amirtu Please comment changes in reference code.

Reply:
@alalek @asmorkalov Sorry for delayed answer, it has taken me some time to understand the context. Here are approximations of partial derivatives being computed with Sobel operator, the scaling is needed to preserve the energy after this filter: power of two due to averaging and differentiation parts of the filter, block size is considered in advance due to windowing of the image for every point, CV_32C1 is (probably) normalized, CV_8U is not. First derivatives in both horizontal and vertical direction then should be scaled, as can be seen in cornerEigenValsVecs() (/imgproc/.../corner.cpp).